modules/ROOT/partials/configuration/iframe_template_callback.adoc

```diff
@@ -0,0 +1,24 @@
+[[iframe_template_callback]]
+== `+iframe_template_callback+`
+
+include::partial$misc/admon-requires-6.1v.adoc[]
+
+This option allows you to specify the function that will return the HTML embed code of the iframe media that you are attempting to insert into {productname}.
+
+*Type:* `+String+`
+
+=== Example: Using `+iframe_template_callback+`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'media',
+  toolbar: 'media',
+  iframe_template_callback: (data) =>
+    `<iframe title="${data.title}" width="${data.width}" height="${data.height}">` +
+    `<source src="${data.source}"${data.sourcemime ? ` type="${data.sourcemime}"` : ''} />\n` +
+    (data.altsource ? `<source src="${data.altsource}"${data.altsourcemime ? ` type="${data.altsourcemime}"` : ''} />\n` : '') +
+    '</iframe>'
```

Comment:
```suggestion
    `<iframe title="${data.title}" width="${data.width}" height="${data.height}" src="${data.source}"></iframe>`
```
`<source>` is not a valid child element for an iframe, so this should be a more accurate example.